br/pkg/restore/data.go

```diff
@@ -292,50 +300,82 @@ func (recovery *Recovery) WaitApply(ctx context.Context) (err error) {
 	return eg.Wait()
 }
 
-// ResolveData a worker pool to all tikv for execute delete all data whose has ts > resolvedTs
-func (recovery *Recovery) ResolveData(ctx context.Context, resolvedTs uint64) (err error) {
+// get the region key range
+func (recovery *Recovery) getRegionKeyRanges() []tikvstore.KeyRange {
```

Comment:
Why not just use 
```
StartKey: 0x00,
EndKey:   {},
```
as key range for whole cluster? The sendFlashback will split key ranges by region cache internally. It's no need to split them here.

Reply:
fix in latest commit